Makefile

```diff
@@ -61,6 +62,10 @@ generate: controller-gen
 # Build all docker images for amd64 and arm64
 build: build-op
 
+# Build the docker image for amd64 and arm64
```

Comment:
Build the docker image for amd64 and arm64
=>
Build amd64/arm64 Fluent Bit container image

Build the docker image for amd64 and arm64
=>
Build amd64/arm64 Fluent Bit Operator container image